src/Progress/LinearProgress.js

```diff
@@ -13,27 +13,27 @@ export const styles = theme => ({
     height: 5,
   },
   primaryColor: {
-    backgroundColor: theme.palette.primary[100],
+    backgroundColor: theme.palette.primary.light,
```

Comment:
The color is too close to `theme.palette.primary.main`. We can't see the difference:
![capture d ecran 2018-01-09 a 08 24 07](https://user-images.githubusercontent.com/3165635/34709201-7c9db5e0-f516-11e7-8300-e7ac2c98d7a6.png)
I think that we need a `lighten()` call.

Reply:
> @oliviertassinari 
> We can use A200 for A100

Yeah, I had my doubts about that. `lighten()` it is.